projects/ngx-d3timeline/src/lib/ngx-d3timeline.component.ts

```diff
@@ -62,6 +60,8 @@ export class NgxD3timelineComponent
   }
 
   @Output() activityDropped = new EventEmitter<Activity>();
+  @Output() hovered = new EventEmitter<Activity>();
```

Comment:
```suggestion
  @Output() activityHovered = new EventEmitter<Activity>();
```

Reply:
This event will also have to fire when a resource is hovered

Reply:
what will its generic type be then? ie: currently its argument is an Activity, if it handles both will it be Activity | Resource ?

Also we are differentiating hover and unhover, which (arguably) are more closely linked then hovering an Activity and hovering a Resource

Reply:
we don't have to specify the generic type. A resource is always a string so could be Activity | string i.e. object or string.